Tests/FunctionalTests/MiscellaneousTests.swift

```diff
@@ -337,6 +337,31 @@ class MiscellaneousTestCase: XCTestCase {
         }
     }
 
+    func testLTO() throws {
+        #if os(macOS)
+        // FIXME: this test requires swift-driver to be installed
+        // Currently swift-ci does not build/install swift-driver before running
+        // swift-package-manager tests which results in this test failing.
+        // See the following additional discussion:
+        // - https://github.com/apple/swift/pull/69696
+        // - https://github.com/apple/swift/pull/61766
+        // - https://github.com/apple/swift-package-manager/pull/5842#issuecomment-1301632685
+        try fixture(name: "Miscellaneous/LTO/SwiftAndCTargets") { fixturePath in
+            do {
+                let output = try executeSwiftBuild(
+                  fixturePath,
```

Comment:
This seems to be using a mix of 4 and 2-space formatting. Additionally, why wrap this in `do`/`catch` if `try fixture(name:)` would be rethrowing the error anyway?

Reply:
I had added `XCTFail("\(error)")` to help with debugging, removing and fixing indentation